torch/csrc/distributed/c10d/ProcessGroup.hpp

```diff
@@ -411,7 +414,7 @@ class TORCH_API ProcessGroup : public torch::CustomClassHolder {
       int /* tag */) {
     TORCH_CHECK(
         false,
-        c10::str("ProcessGroup ", getBackendName(), "does not support send"));
+        c10::str("ProcessGroup ", getBackendName(), " does not support send"));
```

Comment:
Absolutely love the changes, but do you mind moving them to a separate PR (so that we can land it quickly and reduce size of this one)

Reply:
Split out at https://github.com/pytorch/pytorch/pull/84780